### PR TITLE
clipboard button added #99

### PR DESCRIPTION
**Pull request by gunjanmimo**
Signed-off-by: gunjanmimo <gunjan.mtbpaul@gmail.com>

### CODE REVIEW

File: _includes/getting-started.html [hunk 1]
```diff
@@ -51,7 +51,7 @@ <h3 style="margin-left:1em;">Bash user</h3 style="margin-left:1.5em;">
   
               <div id="start-brew" class="highlight-code highlight-source-shell" style="margin-left:1.8em;">
                 <div style="width:auto; height:60px; padding:0px; position:relative;">
-                  <pre><code> $ brew tap layer5io/tap<br /> $ brew install mesheryctl<br /> $ mesheryctl start<br /> 
+                  <pre><code> $ brew tap layer5io/tap<br /> $ kubectl create namespace meshery<br /> $ mesheryctl start<br />
```
Comment (leecalcote) [inline _includes/getting-started.html]: `$ brew tap layer5io/tap` can be removed here.

File: _includes/getting-started.html [hunk 2]
```diff
@@ -60,7 +60,7 @@ <h3 style="margin-left:1em;">Bash user</h3 style="margin-left:1.5em;">
                       border-radius: 2px; margin:0px 0px 880px 0px;"
                     data-clipboard-target="#start-brew"
                     data-clipboard-text="brew tap layer5io/tap
```
Comment (leecalcote) [inline _includes/getting-started.html]: `brew tap layer5io/tap` can be removed here.

File: _includes/getting-started.html [hunk 3]
```diff
@@ -60,7 +60,7 @@ <h3 style="margin-left:1em;">Bash user</h3 style="margin-left:1.5em;">
                       border-radius: 2px; margin:0px 0px 880px 0px;"
                     data-clipboard-target="#start-brew"
```
Comment (leecalcote) [inline _includes/getting-started.html]: Might want to change this to `start-k8s`.

File: _includes/getting-started.html [hunk 4]
```diff
@@ -89,44 +89,93 @@ <h3 style="margin-left:1em;">Kubernetes user</h3 style="margin-left:1.5em;">
             <!-- Kubernetes - Helm -->
             <h4 style="font-size: 1.5em;">Using Helm</h4>
             <p>Helm v3 - run the following:</p>
-            <div class="highlight-code highlight-source-shell">
-              <pre><code>
-  $ git clone https://github.com/layer5io/meshery.git; cd meshery
-  $ kubectl create namespace meshery
-  $ helm install meshery --namespace meshery install/kubernetes/helm/meshery
-              </code></pre>
+            <div id="start-brew" class="highlight-code highlight-source-shell" style="margin-left:1.8em;">
```
Comment (leecalcote) [inline _includes/getting-started.html]: Change class id to `start-helm-v3`

File: _includes/getting-started.html [hunk 5]
```diff
@@ -89,44 +89,93 @@ <h3 style="margin-left:1em;">Kubernetes user</h3 style="margin-left:1.5em;">
...
 
             <p>Helm v2 - run the following:</p>
-              <div class="highlight-code highlight-source-shell">
-              <pre><code>
-  $ git clone https://github.com/layer5io/meshery.git; cd meshery
-  $ kubectl create namespace meshery
-  $ helm template meshery --namespace meshery install/kubernetes/helm/meshery
-              </code></pre>
+              
+            <div id="start-brew" class="highlight-code highlight-source-shell" style="margin-left:1.8em;">
```
Comment (leecalcote) [inline _includes/getting-started.html]: Change class id to `start-helm-v2`

File: _includes/getting-started.html [hunk 6]
```diff
@@ -89,44 +89,93 @@ <h3 style="margin-left:1em;">Kubernetes user</h3 style="margin-left:1.5em;">
...
+              <div style="width:auto; height:60px; padding:0px; position:relative;">
+                <pre><code> $ git clone https://github.com/layer5io/meshery.git; cd meshery<br /> $ kubectl create namespace meshery<br /> $ helm template meshery --namespace meshery install/kubernetes/helm/meshery<br /> 
+                </code></pre>
+                <!-- copy to clipboard -->
+                <a class="btn waves-effect waves-dark white-text darken-2 l5-light-blue z-depth-4"
+                style="position: absolute;top:-14px; right: -1px; border: 0; 
+                    cursor: pointer; background:#00a4ca; width:60px; height:75px; margin-bottom:35px; 
+                    border-radius: 2px; margin:0px 0px 880px 0px;"
+                  data-clipboard-target="#start-brew"
```
Comment (leecalcote) [inline _includes/getting-started.html]: change target to `start-helm-v2`

File: _includes/getting-started.html [hunk 7]
```diff
@@ -89,44 +89,93 @@ <h3 style="margin-left:1em;">Kubernetes user</h3 style="margin-left:1.5em;">
...
+            <div id="start-brew" class="highlight-code highlight-source-shell" style="margin-left:1.8em;">
+              <div style="width:auto; height:60px; padding:0px; position:relative;">
+                <pre><code> $ git clone https://github.com/layer5io/meshery.git; cd meshery<br /> $ kubectl create namespace meshery<br /> $ helm install meshery --namespace meshery install/kubernetes/helm/meshery<br /> 
+                </code></pre>
+                <!-- copy to clipboard -->
+                <a class="btn waves-effect waves-dark white-text darken-2 l5-light-blue z-depth-4"
+                style="position: absolute;top:-14px; right: -1px; border: 0; 
+                    cursor: pointer; background:#00a4ca; width:60px; height:75px; margin-bottom:35px; 
+                    border-radius: 2px; margin:0px 0px 880px 0px;"
+                  data-clipboard-target="#start-brew"
```
Comment (leecalcote) [inline _includes/getting-started.html]: change target to `start-helm-v3`

File: _includes/getting-started.html [hunk 8]
```diff
@@ -89,44 +89,93 @@ <h3 style="margin-left:1em;">Kubernetes user</h3 style="margin-left:1.5em;">
...
+          <div id="start" class="highlight-code highlight-source-shell" style="margin-left:1.8em;">
+            <div style="width:auto; height:30px; padding:0px; position:relative;">
+              <pre><code>   $ git clone https://github.com/layer5io/meshery.git; cd meshery    
+              </code></pre>
+              <!-- copy to clipboard -->
+              <a class="btn waves-effect waves-dark white-text darken-2 l5-light-blue z-depth-4"
+                style="position: absolute;top:-14px; right: -1px; border: 0; 
+                    cursor: pointer; background:#00a4ca; width:60px; height:44px; margin-bottom:35px; 
+                    border-radius: 2px; margin:0px 0px 880px 0px;"
+                data-clipboard-target="#start"
```
Comment (leecalcote) [inline _includes/getting-started.html]: Maybe use `start-k8s-manifests`

File: _includes/getting-started.html [hunk 9]
```diff
@@ -89,44 +89,93 @@ <h3 style="margin-left:1em;">Kubernetes user</h3 style="margin-left:1.5em;">
...
           <!-- Kubernetes - Manifests -->
           <h4 style="font-size: 1.5em;">Using Manifests directly</h4>
           <p>Using Kubernetes, install Meshery on your cluster by cloning the Meshery repo:</p>
-          <div class="highlight-code highlight-source-shell">
+          <!-- <div class="highlight-code highlight-source-shell">
             <pre><code>
   $ git clone https://github.com/layer5io/meshery.git; cd meshery        
             </code></pre>
+          </div> -->
+          <div id="start" class="highlight-code highlight-source-shell" style="margin-left:1.8em;">
```
Comment (leecalcote) [inline _includes/getting-started.html]: Maybe use `start-k8s-manifests`

File: _includes/getting-started.html [hunk 10]
```diff
@@ -89,44 +89,93 @@ <h3 style="margin-left:1em;">Kubernetes user</h3 style="margin-left:1.5em;">
...
+        <!-- <div class="highlight-code highlight-source-shell">
           <pre><code>
   $ kubectl create ns meshery
   $ kubectl -n meshery apply -f deployment_yamls/k8s
           </code></pre>
-          <!-- PROVIDE THE "ON THE MESH" VERSION IN FUTURE DOCUMENTATION
-          # additional step for running on Istio
-          $ kubectl -n meshery apply -f deployment_yamls/istio.yaml -->
+        </div> -->
+        <div id="start" class="highlight-code highlight-source-shell" style="margin-left:1.8em;">
```
Comment (leecalcote) [inline _includes/getting-started.html]: Maybe use `start-k8s-in-cluster`

File: _includes/getting-started.html [hunk 11]
```diff
@@ -89,44 +89,93 @@ <h3 style="margin-left:1em;">Kubernetes user</h3 style="margin-left:1.5em;">
...
+          <div style="width:auto; height:60px; padding:0px; position:relative;">
+            <pre><code> $ git clone https://github.com/layer5io/meshery.git; cd meshery<br /> $ kubectl create namespace meshery<br /> $ helm template meshery --namespace meshery install/kubernetes/helm/meshery<br /> 
+            </code></pre>
+            <!-- copy to clipboard -->
+            <a class="btn waves-effect waves-dark white-text darken-2 l5-light-blue z-depth-4"
+              style="position: absolute;top:-14px; right: -1px; border: 0; 
+                  cursor: pointer; background:#00a4ca; width:60px; height:74px; margin-bottom:35px; 
+                  border-radius: 2px; margin:0px 0px 880px 0px;"
+              data-clipboard-target="#start"
```
Comment (leecalcote) [inline _includes/getting-started.html]: Maybe use `start-k8s-in-cluster`

File: _includes/getting-started.html [hunk 12]
```diff
@@ -89,44 +89,93 @@ <h3 style="margin-left:1em;">Kubernetes user</h3 style="margin-left:1.5em;">
...
+              data-clipboard-text="kubectl create ns meshery
+              kubectl -n meshery apply -f deployment_yamls/k8s ">
+              <img src="/images/buttons/clippy.svg" style="width:20px; margin: 0; position: absolute; top: 50%; left: 54%; transform: translate(-50%, -50%);" />
+            </a>
         </div>
       </div>
+      </div>
     
     <h3 style="margin-left:1em;">Windows user</h3> 
     <p style="margin-left:1.8em;display: inline;">
```
Comment (leecalcote) [inline _includes/getting-started.html]: Will you change Windows from a hyperlink to the documentation to including the download binary instructions directly here? This way, the user doesn't need to be directed to another site for these instructions.

Comment (gunjanmimo) [reply _includes/getting-started.html]: sir I have added

**Pull request merged**